Source/MLX/Documentation.docc/Organization/arithmetic.md

```diff
@@ -73,9 +73,6 @@ Note: the `-` and `/` operators are not able to be linked here.
 - ``MLXArray/+(_:_:)-1rv98``
 - ``MLXArray/+(_:_:)-2vili``
 - ``MLXArray/+(_:_:)-1jn5i``
-- ``MLXArray/-(_:_:)-7frdo``
-- ``MLXArray/-(_:_:)-9mf3``
-- ``MLXArray/-(_:_:)-971j0``
```

Comment:
These are not usable in docs, see also https://github.com/ml-explore/mlx-swift/issues/13